src/styles/layout.sss

```diff
@@ -41,19 +41,17 @@
 .main
     flex: 5 0 60%
     display: flex
-    box-sizing: border-box
+    flex-direction: row
+    overflow: scroll
 
 .main__content
+    overflow: auto
```

Comment:
I believe we only need overflow-y as content should text that is moved to next line in case it is too big?